pulpcore/cli/common/debug.py

```diff
@@ -17,10 +18,34 @@ def debug() -> None:
 
 @debug.command()
 @click.option("--name", required=True)
-@click.option("--min-version")
-@click.option("--max-version")
+@click.option("--min-version", help=_("Succeed only if the installed version is not smaller."))
+@click.option("--max-version", help=_("Succeed only if the installed version is smaller."))
 @pass_pulp_context
 def has_plugin(pulp_ctx: PulpContext, name: str, min_version: str, max_version: str) -> None:
+    """
+    Check whether a specific plugin is installed on the server.
+    """
     available = pulp_ctx.has_plugin(name, min_version, max_version)
     pulp_ctx.output_result(available)
     sys.exit(0 if available else 1)
+
+
+@debug.command()
+@click.option("--seconds", type=int, default=10, help=_("Number of seconds to wait at most."))
+@pass_pulp_context
+def wait_for_pulp(pulp_ctx: PulpContext, seconds: int) -> None:
+    """
+    Repeatedly check for the status of the server to be available.
+    """
```

Comment:
Maybe it's worth to include this functionality in the regular `status` command.

Reply:
I could see that as long as it's not the default behavior.